df.viewscreen.xml

```diff
@@ -1730,55 +1726,62 @@
             <int32_t name='entity' ref-target='historical_entity'/>
             <int32_t name='histfig' refers-to='$global.world.nemesis.all[$]'/>
             <int32_t name='unk_284'/>
-            <int32_t name='unk_288'/>
+            <int32_t name='cultural_identity' ref-target='cultural_identity'/>
             <int16_t name='unk_28c'/>
             <int32_t name='unk_290'/>?
             <int32_t name='unk_294'/>?
             <int16_t name='unk_298'/>
 
-            <static-array count='19' name='attributes'><enum type-name='adventurer_attribute_level'/></static-array>
+            <static-array count='6' name='physical_levels' index-enum='physical_attribute_type'>
+                <enum type-name='adventurer_attribute_level'/>
+            </static-array>
+            <static-array count='13' name='mental_levels' index-enum='mental_attribute_type'>
+                <enum type-name='adventurer_attribute_level'/>
+            </static-array>
             <enum base-type='int16_t' name='status'>
                 <enum-item name='Peasant'/>
                 <enum-item name='Hero'/>
                 <enum-item name='Demigod'/>
             </enum>
 
-            <int32_t name='unk_v40_1a' init-value='-1' since='v0.40.01'/>
+            <enum name='background_type' base-type='int32_t' since='v0.40.01'>
+                <enum-item name='Peasant' value='-1'/>
```

Comment:
It doesn't work because they're in the same class and C++ is annoying when it comes to this (I vaguely remember running into this myself in `viewscreen_titlest`).